src/language/python/bootstrap.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I don't think this file is necessary.  It's targeted specifically to homebrew, which isn't good.  I think it should just be deleted, and the various build / test rules placed in a `make` file.